Scripts/NuGet/EdFi.Installer.WebApi/Install-EdFiOdsWebApi.psm1

```diff
@@ -390,12 +396,18 @@ function Invoke-TransformWebConfigAppSettings {
             $settings = Get-Content $settingsFile | ConvertFrom-Json | ConvertTo-Hashtable
             $settings.ApiSettings.Mode = $Config.InstallType
             $settings.ApiSettings.Engine = $Config.engine
-            $settings.ApiSettings.Features=$Config.WebApiFeatures.Features
-            $settings.ApiSettings.BearerTokenTimeoutMinutes=$Config.WebApiFeatures.BearerTokenTimeoutMinutes
-            $settings.ApiSettings.ExcludedExtensions=$Config.WebApiFeatures.ExcludedExtensions
-            $EmptyHashTable=@{}
-            $mergedSettings = Merge-Hashtables $settings, $EmptyHashTable
-            New-JsonFile $settingsFile $mergedSettings -Overwrite
+            If ( $Config.WebApiFeatures.Features -ne $Null -and $Config.WebApiFeatures.Features -ne '') {
```

Comment:
do we need to check `Features -ne ''` here?

Reply:
No . I removed it . Thanks